charts/scorpio-broker-dist/templates/scorpio-gateway-node-port-svc.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.gateway.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: {{ .Values.gateway.name }}
+  name: scorpio-gateway-node-port
+  namespace: {{ .Release.Namespace }}
+spec:
```

Comment:
This should be configurable. It will fail in environments where nodeport biding is not allowed or 30001 is already used by someone else.

Reply:
We have incorporated the review comment. Please review again